From 801a78f44d60503d19f17f24b8e0464f506e6bf4 Mon Sep 17 00:00:00 2001
From: Yi Xie <imxieyi@users.noreply.github.com>
Date: Sun, 4 May 2025 22:28:19 +0900
Subject: [PATCH] Fix build options

---
 .bazelrc | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/.bazelrc b/.bazelrc
index 86b38a8f41..c0c0db8a23 100644
--- a/.bazelrc
+++ b/.bazelrc
@@ -10,6 +10,7 @@ build --cxxopt='-std=c++17'
 build --per_file_copt="third-party/webrtc/.*\.cpp$","@-std=c++17"
 build --per_file_copt="third-party/webrtc/.*\.cc$","@-std=c++17"
 build --per_file_copt="third-party/webrtc/.*\.mm$","@-std=c++17"
+build --per_file_copt="submodules/AsyncDisplayKit/Source/.*\.mm$","@-Wno-vla-extension"
 build --per_file_copt="submodules/LottieMeshSwift/LottieMeshBinding/Sources/.*\.mm$","@-std=c++17"
 build --per_file_copt="submodules/LottieCpp/lottiecpp/Sources/.*\.mm$","@-std=c++17"
 build --per_file_copt="submodules/LottieCpp/lottiecpp/Sources/.*\.cpp$","@-std=c++17"
@@ -19,8 +20,6 @@ build --per_file_copt="Tests/LottieMetalTest/SoftwareLottieRenderer/Sources/.*\.
 build --per_file_copt="Tests/LottieMetalTest/SoftwareLottieRenderer/Sources/.*\.mm$","@-std=c++17"
 build --per_file_copt="third-party/td/TdBinding/Sources/.*\.mm$","@-std=c++17"
 
-build --swiftcopt=-whole-module-optimization
-
 build --per_file_copt=".*\.m$","@-fno-objc-msgsend-selector-stubs"
 build --per_file_copt=".*\.mm$","@-fno-objc-msgsend-selector-stubs"
 
-- 
2.39.5 (Apple Git-154)

